schema.sql

```diff
@@ -1,5 +1,12 @@
-/* Database schema to keep the structure of entire database. */
+--  Create a database named vet_clinic
+CREATE DATABASE vet_clinic;
 
+-- Create a table named animals with the following columns:
 CREATE TABLE animals (
-    name varchar(100)
+    id INTEGER,
```

Comment:
- [ ] [OPTIONAL] Kindly make the id column not null and autoincremented so you don't have to insert it manually.
```
id int NOT NULL IDENTITY(1,1)
```